apps/studio/components/interfaces/Home/ServiceStatus.tsx

```diff
@@ -193,6 +193,11 @@ export const ServiceStatus = () => {
             docsUrl: 'https://supabase.com/docs/guides/functions/troubleshooting',
             isLoading,
             isSuccess: edgeFunctionsStatus?.healthy,
+            status: edgeFunctionsStatus?.healthy
```

Comment:
any reason not to make `status`, `isSuccess` etc required fields? That would've prevented this afaict

If a service doesn't have data yet, it can fall back to 'unhealthy' / coming up

Reply:
yup makes sense! will update

Reply:
pushed an update!